lib/variables.ts

```diff
@@ -13,3 +13,68 @@ export const distro = {
     id: GLib.get_os_info('ID'),
     logo: GLib.get_os_info('LOGO'),
 };
+
+export const distroIcon = ((): string => {
```

Comment:
I like this but let's make it a 2d array as that's easier to manage and extend. Ex:
```ts
['arch', ''],
['nixos', '']

Reply:
Then we can just do something like
```ts
const distroIcon = distroIconMap.find(distroItem => distroItem[0] === distro.id);

return distroIcon || '';

Reply:
sure thing.